Automated Smoketest: :x:
OS: centos 7.4.1708 (Linux 3.10.0-327.28.3.el7.x86_64)
Sha: a9676c4d29d4910ee196ee57a6f217435dd6d789
Number of Error(s): 56
/mnt/disk1/home/vamosax/smoketest/PR-12538/HPCC-Platform/esp/scm/ws_store.ecm(146) : syntax error H99 : syntax error near }
/mnt/disk1/home/vamosax/smoketest/PR-12538/HPCC-Platform/esp/scm/ws_store.ecm(146) : syntax error H99 : syntax error near }
/mnt/disk1/home/vamosax/smoketest/PR-12538/build/generated/ws_store.hpp:493:5: error: expected unqualified-id before \xe2\x80\x98/\xe2\x80\x99 token
 *** /mnt/disk1/home/vamosax/smoketest/PR-12538/HPCC-Platform/esp/scm/ws_store.ecm(146) syntax error H99 : syntax error near }
/mnt/disk1/home/vamosax/smoketest/PR-12538/build/generated/ws_store.hpp:493:5: error: expected constructor, destructor, or type conversion before \xe2\x80\x98/\xe2\x80\x99 token
/mnt/disk1/home/vamosax/smoketest/PR-12538/build/generated/ws_store.hpp:493:5: error: expected unqualified-id before \xe2\x80\x98/\xe2\x80\x99 token
 *** /mnt/disk1/home/vamosax/smoketest/PR-12538/HPCC-Platform/esp/scm/ws_store.ecm(146) syntax error H99 : syntax error near }
/mnt/disk1/home/vamosax/smoketest/PR-12538/build/generated/ws_store.hpp:493:5: error: expected constructor, destructor, or type conversion before \xe2\x80\x98/\xe2\x80\x99 token
/mnt/disk1/home/vamosax/smoketest/PR-12538/build/generated/ws_store.hpp:493:5: error: expected unqualified-id before \xe2\x80\x98/\xe2\x80\x99 token
 *** /mnt/disk1/home/vamosax/smoketest/PR-12538/HPCC-Platform/esp/scm/ws_store.ecm(146) syntax error H99 : syntax error near }
/mnt/disk1/home/vamosax/smoketest/PR-12538/build/generated/ws_store.hpp:493:5: error: expected constructor, destructor, or type conversion before \xe2\x80\x98/\xe2\x80\x99 token
/mnt/disk1/home/vamosax/smoketest/PR-12538/HPCC-Platform/esp/services/ws_store/ws_storeService.hpp:32:1: error: expected class-name before \xe2\x80\x98{\xe2\x80\x99 token
/mnt/disk1/home/vamosax/smoketest/PR-12538/HPCC-Platform/esp/services/ws_store/ws_storeService.hpp:34:125: error: class \xe2\x80\x98CwsstoreSoapBindingEx\xe2\x80\x99 does not have any field named \xe2\x80\x98CwsstoreSoapBinding\xe2\x80\x99
/mnt/disk1/home/vamosax/smoketest/PR-12538/HPCC-Platform/esp/services/ws_store/ws_storeService.hpp:40:1: error: expected class-name before \xe2\x80\x98{\xe2\x80\x99 token
/mnt/disk1/home/vamosax/smoketest/PR-12538/HPCC-Platform/esp/services/ws_store/ws_storeService.hpp:58:46: error: \xe2\x80\x98IEspCreateStoreRequest\xe2\x80\x99 has not been declared
/mnt/disk1/home/vamosax/smoketest/PR-12538/HPCC-Platform/esp/services/ws_store/ws_storeService.hpp:58:75: error: \xe2\x80\x98IEspCreateStoreResponse\xe2\x80\x99 has not been declared
/mnt/disk1/home/vamosax/smoketest/PR-12538/HPCC-Platform/esp/services/ws_store/ws_storeService.hpp:63:10: error: \xe2\x80\x98bool CwsstoreEx::attachServiceToDali()\xe2\x80\x99 marked override, but does not override
/mnt/disk1/home/vamosax/smoketest/PR-12538/HPCC-Platform/esp/services/ws_store/ws_storeService.hpp:69:10: error: \xe2\x80\x98bool CwsstoreEx::detachServiceFromDali()\xe2\x80\x99 marked override, but does not override
/mnt/disk1/home/vamosax/smoketest/PR-12538/HPCC-Platform/system/jlib/jiface.hpp:278:51: error: cannot call member function \xe2\x80\x98void CSimpleInterfaceOf<INTERFACE>::Link() const [with INTERFACE = CEmptyClass]\xe2\x80\x99 without object
/mnt/disk1/home/vamosax/smoketest/PR-12538/HPCC-Platform/system/jlib/jiface.hpp:279:61: error: cannot call member function \xe2\x80\x98bool CInterface::Release() const\xe2\x80\x99 without object
/mnt/disk1/home/vamosax/smoketest/PR-12538/HPCC-Platform/esp/services/ws_store/ws_storeService.cpp:120:54: error: \xe2\x80\x98IEspCreateStoreRequest\xe2\x80\x99 has not been declared
/mnt/d
 ... ( comment is too long, 64 lines, 10962 bytes, truncated) ... 
[Build error:](https://gist.github.com/d1c9517bc48ac940056001016dce57ad)
Build log [Build log](https://gist.github.com/HPCCSmoketest/d1c9517bc48ac940056001016dce57ad#file-result-19-05-10-11-42-02-log)

```

--- a/esp/scm/ws_store.ecm
+++ b/esp/scm/ws_store.ecm
@@ -142,7 +142,7 @@ ESPrequest CreateStoreRequest
     string Name;
     string Type;
     string Description;
-    bool   UserSpecific(false);
+    [depr_ver("1.01")] bool UserSpecific(false);
 };
 
 ESPresponse [exceptions_inline] CreateStoreResponse

